v3/src/components/web-view/web-view-registration.ts

```diff
@@ -7,7 +7,10 @@ import { ITileModelSnapshotIn } from "../../models/tiles/tile-model"
 import { toV3Id } from "../../utilities/codap-utils"
 import { t } from "../../utilities/translation/translate"
 import { registerV2TileImporter, V2TileImportArgs } from "../../v2/codap-v2-tile-importers"
-import { isV2WebViewComponent, isV2GameViewComponent } from "../../v2/codap-v2-types"
+import { registerV2TileExporter, V2ExportedComponent } from "../../v2/codap-v2-tile-exporters"
+import {
+  isV2WebViewComponent, isV2GameViewComponent, ICodapV2WebViewStorage, ICodapV2WebViewComponent
```

Comment:
```suggestion
  isV2WebViewComponent, isV2GameViewComponent, ICodapV2WebViewComponent
```
I believe `ICodapV2WebViewStorage` is unused now.